boards/hifive1/src/main.rs

```diff
@@ -38,7 +38,7 @@ static mut APP_MEMORY: [u8; 5 * 1024] = [0; 5 * 1024];
 /// Dummy buffer that causes the linker to reserve enough space for the stack.
 #[no_mangle]
 #[link_section = ".stack_buffer"]
-pub static mut STACK_MEMORY: [u8; 0x1000] = [0; 0x1000];
+pub static mut STACK_MEMORY: [u8; 0x800] = [0; 0x800];
```

Comment:
Was this intentionally included in this PR?

Reply:
Yes -- otherwise it won't compile.